src/core/enums.hpp

```diff
@@ -143,6 +143,14 @@ namespace big
 		UNK2                 = 1 << 16
 	};
 
+	enum eKnockOffVehicle
```

Comment:
You have to strongly type the enum to the parameter instance type or class keyword, or else you are just creating an enum type.

Reply:
Oh, thanks for explaining.
And shouldn't this enum be in `/gta/enums.hpp`?

Reply:
If the enumeration comes from GTA itself it should.